### PR TITLE
Update expo permissions to comply with google policies

### PR DESCRIPTION
**Pull request by sseerrggii**
Changes in app.json for compy with google policies to reduce de permissions in the apps when are not needed

https://docs.expo.io/versions/v31.0.0/workflow/configuration

>To use ONLY the following minimum necessary permissions and none of the extras supported
      by Expo, set "permissions" to [].

Also: 
* Updated Expo version 31.0.0 (In my first approach, I thought update expo will resolve the problem, when I discovered that didn't help I decided keep this version of expo in order to keep the repo up to date)
* I deleted icon in push notifications

### CODE REVIEW

**Comment by sseerrggii** [comment]
This build is for testing and release candidate https://expo.io/builds/8cdc8c61-c412-42d2-9212-bb897955795c

**Comment by sseerrggii** [comment]
We can use this build for testing and as new release candidate 

https://expo.io/builds/8cdc8c61-c412-42d2-9212-bb897955795c

**Review by enricostano — APPROVED** [review]

File: app.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "TimeOverflow",
     "description": "A time banking system.",
     "slug": "time-overflow",
-    "sdkVersion": "24.0.0",
+    "sdkVersion": "31.0.0",
```
Comment (mllocs) [inline app.json:6]: Hope there's no breaking changes :D

**Comment by sseerrggii** [comment]
Tested :green_heart: 

Session opened in android for 5 days, may be this also fix the problems with session :man_shrugging:

**Pull request merged**